pkg/sqlcmd/commands_test.go

```diff
@@ -226,6 +226,20 @@ func TestErrorCommand(t *testing.T) {
 	}
 }
 
+func TestOnErrorCommand(t *testing.T) {
+	s, buf := setupSqlCmdWithMemoryOutput(t)
+	defer buf.Close()
+	file, err := os.CreateTemp("", "sqlcmderr")
+	assert.NoError(t, err, "os.CreateTemp")
+	defer os.Remove(file.Name())
+
+	err = onerrorCommand(s, []string{""}, 1)
+	assert.EqualError(t, err, InvalidCommandError("ON ERROR", 1).Error(), "onerrorCommand with empty file name")
+
+	err = runSqlCmd(t, s, []string{":ONERROR ignore", "printtgit N'message'", "SELECT @@versionn", "GO"})
```

Comment:
>ignore", [](http://example.com/codeflow?start=42&length=8)

the test should also verify that using the other argument changes the behavior.

Reply:
Added